platform/ios/src/MGLMapView.mm

```diff
@@ -364,7 +368,11 @@ + (void)initialize
 
 - (nonnull NSURL *)styleURL
 {
-    NSString *styleURLString = @(_mbglMap->getStyle().getURL().c_str()).mgl_stringOrNilIfEmpty;
+    if (!_mbglMap) {
```

Comment:
The brace style in `MGLMapView` is to place them on new lines:

```
if (blah)
{
    ...
}
```

Reply:
Whoa. Ok, we have a few inconsistencies in the SDK. That's for another day ;)

Reply:
Yeah, the iOS-specific code started out with a brace-on-new-line style (as well as a couple other idiosyncrasies), but the rest of the Objective-C codebase (including platform/darwin/ and platform/macos/) generally uses a more Xcode-default style that puts braces on the same line as other code. Over time, that has caused the iOS-specific code to get a bit inconsistent. I personally try not to get worked up over whitespace, but some spring cleaning might eventually be needed. (PRs consistifying coding style tend to create lots of conflicts in outstanding PRs, so they can be a bit disruptive.)